### PR TITLE
[KED-1040] Modify Spaceflights tutorial to support modular pipelines

### PR DESCRIPTION
**Pull request by lorenabalan**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking “Submit Pull Request”:

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Motivation and Context
Address step 4 out of https://jira.quantumblack.com/browse/KED-1040
Copy-pasted the contents of `kedro_cli.py` from a newly-created Kedro project with version `0.15.4`.

## How has this been tested?
Kedro run locally works

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Assigned myself to the PR
- [ ] Added tests to cover my changes

### CODE REVIEW

File: kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def create_pipeline(**kwargs):
+    return Pipeline(
+        [
+            node(
+                preprocess_companies,
+                "companies",
+                "preprocessed_companies",
+                name="preprocess1",
```
Comment (yetudada) [inline kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py]:
```suggestion
                name="preprocessing_companies",
```

File: kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            ),
+            node(
+                preprocess_shuttles,
+                "shuttles",
+                "preprocessed_shuttles",
+                name="preprocess2",
```
Comment (yetudada) [inline kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py]:
```suggestion
                name="preprocessing_shuttles",
```

File: kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+from kedro.pipeline import Pipeline, node
+
+from .nodes import create_master_table, preprocess_companies, preprocess_shuttles
+
+
+def create_pipeline(**kwargs):
+    return Pipeline(
+        [
+            node(
+                preprocess_companies,
```
Comment (yetudada) [inline kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py]: Also, we're going to make this change for the docs but could we be a bit more explicit about the `Node` arguments e.g. `func=preprocess_companies`, `inputs=companies` for all nodes in both pipelines?

Comment (lorenabalan) [reply kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py]: Yes great point!

Comment (lorenabalan) [reply kedro-tutorial/src/kedro_tutorial/pipelines/data_engineering/pipeline.py]: Fixed.

**Comment by lorenabalan** [comment]
> Thanks for updating it! 👍
> 
> Could you also update from 0.15.0 to 0.15.4 in README.md in https://github.com/quantumblacklabs/kedro-examples/tree/master/kedro-tutorial ??

Good catch, fixed!

**Pull request merged**